### PR TITLE
ci: use MOJO format with austin profiles

### PR DESCRIPTION
**Pull request by P403n1x87**
## Description

This change uses the more compact MOJO binary output format from Austin to reduce the size of the artifacts. It also makes use of the resolve tool available from the austin-python package, since the same util is no longer available from the devel branch.

## Checklist
- [x] Add additional sections for `feat` and `fix` pull requests.
- [x] [Library documentation](https://github.com/DataDog/dd-trace-py/tree/1.x/docs) and/or [Datadog's documentation site](https://github.com/DataDog/documentation/) is updated. Link to doc PR in description.

## Reviewer Checklist
- [x] Title is accurate.
- [x] Description motivates each change.
- [x] No unnecessary changes were introduced in this PR.
- [x] Avoid breaking [API](https://ddtrace.readthedocs.io/en/stable/versioning.html#interfaces) changes unless absolutely necessary.
- [x] Tests provided or description of manual testing performed is included in the code or PR.
- [x] Release note has been added for fixes and features, or else `changelog/no-changelog` label added.
- [x] All relevant GitHub issues are correctly linked.
- [x] Backports are identified and tagged with Mergifyio.

### CODE REVIEW

**Review by Kyle-Verhoog — APPROVED** [review]

File: scripts/profiles/django-simple/run.sh [hunk 1]
```diff
@@ -5,7 +5,7 @@ set -eu
 PREFIX=${1}
 
 AUSTIN_INTERVAL=200  # usec
-AUSTIN_EXPOSURE=5  # sec
+AUSTIN_EXPOSURE=3  # sec
```
Comment (Kyle-Verhoog) [inline scripts/profiles/django-simple/run.sh:8]: seems unrelated?

**Review by Kyle-Verhoog — APPROVED** [review]

File: scripts/profiles/django-simple/run.sh (hunk 1 above)
Comment (P403n1x87) [reply scripts/profiles/django-simple/run.sh:8]: I was noticing timeouts, which seemed to be caused by the resolve step taking too long. I'll try to revert this and see if that was not just a fluke. Otherwise, if I change this again it's because of this reason 🙁

**Review by Kyle-Verhoog — APPROVED** [review]

**Review by majorgreys — APPROVED** [review]
